pyslurm/slurm.pxd

```diff
@@ -705,99 +858,97 @@ cdef extern from 'slurm/slurm.h' nogil:
 
     ctypedef power_mgmt_data power_mgmt_data_t
 
-    ctypedef struct job_descriptor:
-        char *account
-        char *acctg_freq
-        char *admin_comment
-        char *alloc_node
+    cdef struct job_descriptor:
+        char* account
```

Comment:
You prefer the pointer denoted on the type and not the variable?

Reply:
It doesn't matter to me. You can change that.